manticore/core/state.py

```diff
@@ -310,6 +310,8 @@ def solve_one(self, expr):
         :return: Concrete value
         :rtype: int
         '''
+        if not issymbolic(expr):
```

Comment:
`solver.get_value` should already handle this case. is the intention to give `state.solve_one(not_symbolic)` and `solver.get_value(cs, not_symbolic)` different behaviors?